src/components/promotion/PromotionDetails.tsx

```diff
@@ -98,9 +97,6 @@ export default function PromotionDetails({
           <HeartOutlined style={styles.heart} />
         </Col>
       </Row>
-      <Row>
-        <Title style={styles.restaurantName}>{restaurantName}</Title>
-      </Row>
```

Comment:
We are no longer storing the user inputted restaurant names in the BE. This should be in the restaurant card ideally

Reply:
Oh I see... I think we could replace this with a call to get the name via the `use-places-autocomplete` library once the upload promotions PR is merged in

Reply:
Oh because we're not storing the restaurant name in the BE anymore, but I think it'd probably be good to still display the restaurant name here because it prevents users from having to open up the restaurant card to see the name

Reply:
I see, yeah I agree it's important to add the restaurant name here, but this might be difficult to do since when we are loading all the promotions, we'd need to get all the restaurant names for all promotions

Reply:
I think for now, I can leave it uncommented and create a ticket once we have the information available. Because this would mean when we call GET /promotions, we'd need to call GET restuarantDetails for each promotion